CHANGELOG.md

```diff
@@ -1,10 +1,4 @@
-## 2.33.0-rc.1 / 2022-01-19
-
-* [ENHANCEMENT] UI: Debounce search bar. #10157
```

Comment:
Why isn't this included in the final changelog? I think it's an independent change.

Reply:
it's not, it's part of  #10131 and #10103

Reply:
Oh ok

Reply:
Yeah, so essentially, if we further enhance features in a RC that were just added with this release, we do not list the feature and its enhancement separately, but just the feature.